### PR TITLE
added all tags

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: joku/cogs/tags.py [hunk 1]
```diff
@@ -72,6 +72,19 @@ def __init__(self, bot: Jokusoramame):
...
+        """
+        Shows all the tags for the current server
+        """
+        # looks kinda bleak but i try my best *shrug*
+        server_tags = await self.bot.rethinkdb.all_tags(ctx.message.server)
+        if not server_tags:
+            await self.bot.say("This server has no tags")
+            return
+
+        await self.bot.say(", ".join([x["name"] for x in server_tags]))
```
Comment (Fuyukai) [inline joku/cogs/tags.py]: Name should be surrounded with ``, as it looks better.

File: joku/cogs/tags.py [hunk 2]
```diff
@@ -72,6 +72,19 @@ def __init__(self, bot: Jokusoramame):
...
 
     @tag.command(pass_context=True)
+    async def all(self, ctx):
+        """
+        Shows all the tags for the current server
+        """
+        # looks kinda bleak but i try my best *shrug*
+        server_tags = await self.bot.rethinkdb.all_tags(ctx.message.server)
+        if not server_tags:
+            await self.bot.say("This server has no tags")
```
Comment (Fuyukai) [inline joku/cogs/tags.py:81]: Full stop at the end.

File: joku/rethink.py [hunk 3]
```diff
@@ -79,6 +79,22 @@ def __init__(self, bot):
 
         return tag
 
+    async def all_tags(self, server: discord.Server):
```
Comment (Fuyukai) [inline joku/rethink.py]: Should be `get_all_tags_for_server` or similar.

**Review by Fuyukai — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
aye aye

**Pull request closed without merging**